package/gluon-neighbour-info/src/gluon-neighbour-info.c

```diff
@@ -35,6 +35,9 @@
 #include <string.h>
 #include <time.h>
 
+char *recvbuffer;
+size_t recvbuffer_len;
```

Comment:
I don't see why this needs to be global - just pass pointers to the functions that need the buffer.

And if you need global variables, always mark them as 'static' to avoid adding them to the dynamic symbol table.

Reply:
done